docs/proxy/datacenter_proxy/examples.md

```diff
@@ -13,13 +13,11 @@ paths:
 
 This page contains code examples for connecting to [datacenter proxies]({{@link proxy/datacenter_proxy.md}}) using [Apify Proxy](https://apify.com/proxy).
 
-If you are building your own Apify [actor]({{@link actors.md}}), below are [examples](#using-the-apify-sdk) specific to the [Apify SDK](https://sdk.apify.com).
-
 See the [connection settings]({{@link proxy/connection_settings.md}}) page for connection parameters.
 
 ## [](#using-the-apify-sdk) Using the Apify SDK
 
-If you're developing an actor using the [Apify SDK](https://sdk.apify.com), you can use Apify Proxy in:
+If you are developing your own Apify [actor]({{@link actors.md}}) using the [Apify SDK](https://sdk.apify.com), you can use Apify Proxy in:
```

Comment:
Missing `CheerioCrawler`. I'm not saying that we necessarily need an example, but it's weird for it to be left out, when everything else is there.